addons/rivet/api/rivet_packages.gd

```diff
@@ -0,0 +1,97 @@
+	## lobby. Players can still join using the /join endpoint (this can be disabled
+	## by the developer by rejecting all new connections after setting the lobby
+	## to closed). Does not shutdown the lobby.
+	##
+	## [url]{https://rivet.gg/docs/matchmaker/api/lobbies/set-closed}[/url]
+	func setClosed(body: Dictionary = {}) -> _RivetResponse:
+		return await Rivet.PUT("matchmaker/lobbies/set_closed", body).wait_completed()
+
+	## Creates a custom lobby.
+	##
+	## [url]{https://rivet.gg/docs/matchmaker/api/lobbies/create}[/url]
+	func create(body: Dictionary = {}) -> _RivetResponse:
+		return await Rivet.POST("matchmaker/lobbies/create", body).wait_completed()
+
+	## Lists all open lobbies.
+	##
+	## [url]{https://rivet.gg/docs/matchmaker/api/lobbies/list}[/url]
+	func list(body: Dictionary = {}) -> _RivetResponse:
+		return await Rivet.GET("matchmaker/lobbies/list", body).wait_completed()
+
+	## 
+	##
+	## [url]{https://rivet.gg/docs/matchmaker/api/lobbies/set-state}[/url]
+	func setState(body: Dictionary = {}) -> _RivetResponse:
+		return await Rivet.PUT("matchmaker/lobbies/state", body).wait_completed()
+
+	## 
+	##
+	## [url]{https://rivet.gg/docs/matchmaker/api/lobbies/get-state}[/url]
+	func getState(body: Dictionary = {}) -> _RivetResponse:
+		return await Rivet.GET("matchmaker/lobbies/state", body).wait_completed()
+
+## Player
+## @experimental
+class Player:
+	## Validates the player token is valid and has not already been consumed then
+	## marks the player as connected.
+	## 
+	## [url]{https://rivet.gg/docs/matchmaker/api/players/connected}[/url]
+	func connected(body: Dictionary = {}) -> _RivetResponse:
+		return await Rivet.POST("matchmaker/players/connected", body).wait_completed()
+
+	## Marks a player as disconnected. # Ghost Players.
+	##
+	## [url]{https://rivet.gg/docs/matchmaker/api/players/disconnected}[/url]
+	func disconnected(body: Dictionary = {}) -> _RivetResponse:
+		return await Rivet.POST("matchmaker/players/disconnected", body).wait_completed()
+
+	## Gives matchmaker statistics about the players in game.
+	##
+	## [url]{https://rivet.gg/docs/matchmaker/api/players/statistics}[/url]
+	func getStatistics(body: Dictionary = {}) -> _RivetResponse:
+		return await Rivet.GET("matchmaker/players/statistics", body).wait_completed()
+
+class Region:
+	## Returns a list of regions available to this namespace.
+	## Regions are sorted by most optimal to least optimal. 
+	## The player's IP address is used to calculate the regions' optimality.
+	##
+	## [url]{https://rivet.gg/docs/matchmaker/api/regions/list}[/url]
+	func list(body: Dictionary = {}) -> _RivetResponse:
+		return await Rivet.GET("matchmaker/regions", body).wait_completed()
+
+## Matchmaker
+## @experimental
+## @tutorial: https://rivet.gg/docs/matchmaker
+class Matchmaker:
+	static var lobby: Lobby = Lobby.new()
+	static var player: Player = Player.new()
+	static var region: Region = Region.new()
```

Comment:
The convention I've been using is to use the plural for the API path (i.e. `lobbies.find`). See https://rivet.gg/docs/matchmaker/api/lobbies/find

I forgot where I used this convention from, but would be good to keep it consistent for now imo.

Reply:
Didn't notice this when I was writing this, good catch :)